models/AppState.py

```diff
@@ -146,28 +156,28 @@ def minimumOrderQuote(self):
             quote = float(self.account.getBalance(self.app.getQuoteCurrency()))
             base_min = float(product["base_min_size"])
 
-            if (quote / price) < base_min:
-                sys.tracebacklimit = 0
-                raise Exception(
-                    f'Insufficient Quote Funds! (Actual: {"{:.8f}".format((quote / price))}, Minimum: {base_min})'
-                )
-
         elif self.app.getExchange() == 'kucoin':
             resp = self.api.authAPI('GET', f'api/v1/symbols')
             product = resp[resp['symbol'] == self.app.getMarket()]
             if len(product) == 0:
                 sys.tracebacklimit = 0
                 raise Exception(f'Market not found! ({self.app.getMarket()})')
 
+            ticker = self.authAPI("GET", f"api/v1/market/orderbook/level1?symbol={self.app.getMarket()}")
 
-            price = self.app.getTicker(self.app.getMarket())[1]
+            price = float(ticker["price"])
             quote = float(self.account.getBalance(self.app.getQuoteCurrency()))
             base_min = '{:f}'.format(float(product['baseMinSize']))
 
-            if (quote / price) < float(base_min):
-                sys.tracebacklimit = 0
-                raise Exception(f'Insufficient Quote Funds! (Actual: {"{:.8f}".format((quote / price))}, Minimum: {base_min})'
-                )
+        if (quote / price) < base_min:
+            if self.app.disableinsufficientfunds:
```

Comment:
Should this be a "if not"?

Reply:
the variable is to disable raising the exception and enable the logging so if disableinsufficientfunds is true it will log a warning otherwise it will continue as normal ie raising the exception.

maybe I should rename the variable to enableinsufficientfundslogging?